.github/workflows/ci.yml

```diff
@@ -0,0 +1,104 @@
+name: Release and Update Manifests
```

Comment:
ci.yml seems like the wrong name? Maybe release_and_publish or trigger_deploy.yml, or maybe just deploy.yml?